src/utils/puns.ts

```diff
@@ -24,7 +29,11 @@ const WARNING_PUNS = [
   'A bit pitchy, but tuna-ble.',
   'Do you sea the problem?',
   'You are skating on fin ice.',
-  'You should mullet over.'
+  'You should mullet over.',
+  'You might need some Vitamin Sea',
+  'Maybe you should scale back',
+  'Whale, whale, whale, what happened here?',
```

Comment:
I really like this one, but it is 4 characters too long.

Reply:
fixed